### PR TITLE
Entity = Export: Input (COPY)

### PR DESCRIPTION
**Pull request by Marina-sukhova**
Entity = Export: Input (COPY)
https://trello.com/c/W0V1OVoU/337-entity-export-input-copy

### CODE REVIEW

File: src/test/java/EntityExportTest.java [hunk 1]
```diff
@@ -221,13 +227,60 @@ public void viewTest() throws InterruptedException {
...
+        WebElement thirdField = driver.findElement(By.xpath("//*[@name='entity_form_data[int]']"));
+        ProjectUtils.click(driver, thirdField);
+        thirdField.sendKeys(exportString);
+
+        WebElement button = driver.findElement(By.xpath("//button[@class='btn btn-warning']"));
+        ProjectUtils.click(driver, button);
+
+        WebElement error = driver.findElement(By.xpath("//div[@id='pa-error']"));
+        error.getText();
+        Assert.assertTrue(true, "Error saving entity");
```
Comment (Stan575) [inline src/test/java/EntityExportTest.java:258]: assertTrue(true) will always pass, you know :D 
there's a bug (not fixed yet) everybody knows about, can you wrap it in a different way?

**Review by Stan575 — CHANGES_REQUESTED** [review]

File: src/test/java/EntityExportTest.java [hunk 2]
```diff
@@ -221,13 +227,60 @@ public void viewTest() throws InterruptedException {
...
         WebElement tableIntField = driver.findElement(By.xpath("//tbody/tr[1]/td[4]"));
-        Assert.assertEquals(tableIntField.getText(),"124");
+        Assert.assertEquals(tableIntField.getText(), "124");
         WebElement tableDecimalField = driver.findElement(By.xpath("//td[contains(text(),'34.56')]"));
-        Assert.assertEquals(tableDecimalField.getText(),"34.56");
+        Assert.assertEquals(tableDecimalField.getText(), "34.56");
+    }
+
+    @Test
+    //Negative test String to Int Field
```
[the remark this answers is not in the document]
Comment (viktoriia1975) [reply src/test/java/EntityExportTest.java:240]: Я не вижу еще ссылку на Трелло задачу. И плюс тест view изменился. Посмотри.

File: src/test/java/EntityExportTest.java (hunk 1 above)
Comment (Stan575) [reply src/test/java/EntityExportTest.java:258]: it's actually expected behavior at this time, not a bug

**Review by Stan575 — APPROVED** [review]
Nicely done!

File: src/test/java/EntityExportTest.java [hunk 3]
```diff
@@ -221,13 +227,30 @@ public void viewTest() throws InterruptedException {
...
+        ProjectUtils.click(driver, export);
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[contains (text(), 'create_new_folder')]"))).click();
+        WebElement thirdField = driver.findElement(By.xpath("//*[@name='entity_form_data[int]']"));
+        ProjectUtils.click(driver, thirdField);
+        thirdField.sendKeys(exportString);
+        WebElement button = driver.findElement(By.xpath("//button[@class='btn btn-warning']"));
+        ProjectUtils.click(driver, button);
+        WebElement error = driver.findElement(By.xpath("//div[@id='pa-error']"));
+        String errorText = error.getText();
+        Assert.assertEquals("Error saving entity", errorText);
```
Comment (Stan575) [inline src/test/java/EntityExportTest.java:253]: public static void assertEquals(String actual, String expected)
assertEquals accepts actual result as first argument, and expected result as second argument, you can swap them to use it right way

**Review by Stan575 — COMMENTED** [review]
Added a note on line 253

File: src/test/java/EntityExportTest.java (hunk 3 above)
Comment (Marina-sukhova) [reply src/test/java/EntityExportTest.java:253]: Got you!

**Comment by Stan575** [comment]
@Marina-sukhova also you need to add Trello task

**Review by PeterKuzyk — APPROVED** [review]

**Pull request merged**